src/LoRa_APRS_Tracker.cpp

```diff
@@ -131,7 +138,28 @@ void setup() {
   batteryVoltage = String(getBattVoltage());
   // EEPROM.begin(512);
 }
-
+void telemetry() {
+  pinMode(Vext, OUTPUT);
+  digitalWrite(Vext, LOW);
+  Serial.begin(115200);
+  if (!bmp.begin()) {
```

Comment:
If no BMP connected we would spam the serial console.
Check for WEATHER_DATA?

Reply:
Yes, that would seem more appropriate